site/docs/features/intro.md

```diff
@@ -0,0 +1,107 @@
+# Introduction
+
+Nessie is an OSS service and libraries that enable you to maintain multiple versions 
+of your data and leverage Git-like Branches & Tags for your Data Lake. Nessie enhances the following 
+table formats with version control techniques:
+
+* Apache Iceberg Tables ([more](../tables/iceberg.md))
+* Delta Lake Tables ([more](../tables/deltalake.md))
+* Hive Metastore Tables ([more](../tables/hive.md))
+* SQL Views ([more](../tables/views.md))
+
+## Basic Concepts
+
+Nessie is heavily inspired by Git. The main concepts Nessie exposes map directly to 
+[Git concepts](https://git-scm.com/book/en/v2). In most cases, you simply need to replace 
+references of files and directories in Git with Tables in Nessie. The primary concepts in Nessie are:
+ 
+* Commit: Consistent snapshot of all tables at a particular point in time
+* Branch: Human-friendly reference that a user can add commits to.
+* Tag: Human-friendly reference that points to a particular commit.
+* Hash: Hexadecimal string representation of a particular commit
+
+Out of the box, Nessie starts with a single branch called `main` that points to the 
+beginning of time. A user can immediately start adding tables to that branch. For example 
+(in pseudo code):
+
+```
+$ create t1
+...
+$ insert 2 records into t1
+...
+$ create t2
+...
+$ insert 2 records into t2
+...
+```
+
+A user can then use the Nessie CLI to view the history of the main branch. You'll see 
+that each operation in Spark was automatically recorded as a commit within Nessie:
```

Comment:
Is Spark important here? I don't think so? Maybe just leave our referring to it and talk about a generic "operation"?

Reply:
Yup, makes sense to "remove" Spark here